app/models/movie.rb

```diff
@@ -0,0 +1,6 @@
+class Movie < ApplicationRecord
+  has_many :directions
+  has_many :directors, through: :directions
+  validates :original_title, presence: true
+  validates :directors, presence: true
```

Comment:
Does this work? 👀 

If not, maybe we should try [validates_length_of](https://apidock.com/rails/v6.0.0/ActiveRecord/Validations/ClassMethods/validates_length_of).

Reply:
It most certainly does! 🤓